### PR TITLE
Step 1 to 6

### PR DESCRIPTION
**Pull request by ldeluigi**
There are obvious conflicts

### CODE REVIEW

**Comment by AndreaCardiota** [comment]
Javadoc is missing, please add it.

**Comment by ldeluigi** [comment]
Done

File: src/main/java/Delu.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+public class Delu {
+
+    /**
+     * Javadoc is my passion!
+     * This is the main.
+     */
+    public static void main(String[] args) {
+        System.out.println("Hello World!");
+    }
+}
```
Comment (DanySK) [inline src/main/java/Delu.java:13]: File should end with newline

File: src/main/java/Delu.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+/**
+ * A super useful class.
+*/
+public class Delu {
+
+    /**
+     * Javadoc is my passion!
+     * This is the main.
```
Comment (DanySK) [inline src/main/java/Delu.java:8]: Missing @param annotation

File: src/main/java/Delu.java (hunk 1 above)
Comment (ldeluigi) [reply src/main/java/Delu.java]: Done in https://github.com/AndreaCardiota/pps-ci-lab/pull/2/commits/3d5d208368a5bc4c843a8726d1ea318524f9170e

File: src/main/java/Delu.java (hunk 2 above)
Comment (ldeluigi) [reply src/main/java/Delu.java:8]: Fixed in https://github.com/AndreaCardiota/pps-ci-lab/pull/2/commits/dae0514428826fcac4114310d12ff019c5366c15

**Pull request merged**